users/ishtob/ishtob.c

```diff
@@ -43,6 +44,22 @@ bool process_record_secrets(uint16_t keycode, keyrecord_t *record) {
   return true;
 }
 
+
+bool muse_mode = false;
+uint8_t last_muse_note = 0;
+uint16_t muse_counter = 0;
+uint8_t muse_offset = 70;
+uint16_t muse_tempo = 50;
+
+#ifdef AUDIO_ENABLE
```

Comment:
The layer floats aren't needed, actually: 
https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer

Reply:
had to keep the plover floats as its called in a function later on, rest are removed